update default values for RocksDB config (#14129)
pingcap/docs: tikv-configuration-file.md

### `capacity`

+ The size of the shared block cache.

> + Default value: 45% of the size of total system memory when `storage.engine="raft-kv"`, 30% of the size of total system memory when `storage.engine="partitioned-raft-kv"`

Reviewer: ```suggestion
+ Default value:

    + When `storage.engine="raft-kv"`, the default value is 45% of the size of total system memory.
    + When `storage.engine="partitioned-raft-kv"`, the default value is 30% of the size of total system memory.

```

### `max-total-wal-size`

+ The maximum RocksDB WAL size in total, which is the size of `*.log` files in the `data-dir`.

> + Default value: `"4GB"` when `storage.engine="raft-kv"`, `1` when `storage.engine="partitioned-raft-kv"`

Reviewer: ```suggestion
+ Default value:

    + When `storage.engine="raft-kv"`, the default value is `"4GB"`.
    + When `storage.engine="partitioned-raft-kv"`, the default value is `1`, which means that the WAL is disabled.
```

Reviewer: https://github.com/pingcap/docs-cn/pull/14598/files#r1275870175

### `stats-dump-period`

+ The interval at which statistics are output to the log.

> + Default value: `"10m"` when `storage.engine="raft-kv"`, `"0"` when `storage.engine="partitioned-raft-kv"`

Reviewer: ```suggestion
+ Default value:

    + When `storage.engine="raft-kv"`, the default value is `"10m"`.
    + When `storage.engine="partitioned-raft-kv"`, the default value is `"0"`.
```

> ### `max-total-wal-size`

Reviewer 2: 中文版这部分没有更新

>
> This feature is experimental. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.

> + Specifies the total memory limit of `memtable` for all RocksDB instances in a single TiKV. Zero stands for no limit.
> + Default value: Zero when `storage.engine="raft-kv"`, 20% of the size of total system memory when `storage.engine="partitioned-raft-kv"`

Reviewer: ```suggestion
+ Specifies the total memory limit of `memtable` for all RocksDB instances in a single TiKV. `0` means no limit.
+ Default value:

    + When `storage.engine="raft-kv"`, the default value is `0`, which means no limit.
    + When `storage.engine="partitioned-raft-kv"`, the default value is 20% of the size of total system memory.

```

    - `3`: Can be read by TiKV v2.1 and later versions. Changes the encoding of the keys in index blocks.
    - `4`: Can be read by TiKV v3.0 and later versions. Changes the encoding of the values in index blocks.
    - `5`: Can be read by TiKV v6.1 and later versions. Full and partitioned filters use a faster and more accurate Bloom filter implementation with a different schema.

> + Default value: `2` when `storage.engine="raft-kv"`, `5` when `storage.engine="partitioned-raft-kv"`

Reviewer: ```suggestion
+ Default value:

    + When `storage.engine="raft-kv"`, the default value is `2`.
    + When `storage.engine="partitioned-raft-kv"`, the default value is `5`.
```
